crates/mockiato-codegen/src/generate.rs

```diff
@@ -26,43 +33,60 @@ pub(crate) struct GenerateMockOptions {
     pub(crate) force_static_lifetimes: bool,
 }
 
+#[cfg_attr(feature = "debug-impls", derive(Debug))]
```

Comment:
What does "debug-impls" do?

Reply:
It enables debug impls 😀. It also enables debug impls for `syn` types.